fava/templates/source.html

```diff
@@ -15,7 +15,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for source_file in api.source_files() %}
+                {% for source_file in api.source_files()|sort %}
```

Comment:
What I've been noticing is that the order in which they're displayed (for me) is pretty random between restarts. Definitely not the order in which they're specified in the main source file. (I've got my files split by year and load them in order, yet in the UI they're random)